msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties

```diff
@@ -816,8 +816,8 @@ move_topics=topics
 move_cancel=Cancel
 move_move=Move Thread(s) to Selected Topic
 move_thread=Move Thread(s)
-move_thread_info1= The following conversation(s) will be moved from
-move_thread_info2= to the topic selected below.
+move_thread_info1= The following conversation(s) will be moved from\u0020
```

Comment:
Can you revert these changes? You're changing the default locale.

Reply:
done